app/forms/openid_connect_authorize_form.rb

```diff
@@ -68,9 +69,9 @@ def loa3_requested?
 
   private
 
-  def parse_acr_values(acr_values)
-    return [] if acr_values.blank?
-    acr_values.split(' ').compact & Saml::Idp::Constants::VALID_AUTHN_CONTEXTS
+  def parse_to_values(str, possible_values)
```

Comment:
in each case that this method is used, it isn't *just* a string, it's the value from a parameter. Perhaps `param_value` would clarify?